conans/client/outputers/cli.py

```diff
@@ -0,0 +1,17 @@
+# coding=utf-8
+from conans.client.output import Color
+from conans.client.outputers.base_outputer import BaseOutputer
+from conans.client.outputers.formats import OutputerFormats
+
+
+@OutputerFormats.register("cli")
```

Comment:
I am not convinced about the registration per class. Not all command/methods/apis will have all formats, so most like the mapping needs to be per command/method. I would probably try to keep it very simple, explicit plain implementation, and automate the registration when we have a couple of patterns.